manifests/script-lab-edge.olk.xml

```diff
@@ -0,0 +1,330 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Also, please make sure that you copy the new app domains from the new manifests.  Better yet, do a diff, and make sure other things didn't change.